OpenEdXMobile/res/layout/fragment_learn.xml

```diff
@@ -1,49 +1,73 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <LinearLayout
-        android:id="@+id/ll_learn_selection"
-        android:layout_width="wrap_content"
-        android:layout_height="@dimen/learn_selection_width"
-        android:layout_gravity="center"
-        android:layout_marginStart="@dimen/container_padding"
-        android:layout_marginEnd="@dimen/container_padding"
-        android:background="@color/white"
-        android:gravity="center_vertical"
-        android:orientation="horizontal"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        tools:ignore="UselessParent">
-
-        <TextView
-            android:id="@+id/tv_selected_item"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:fontFamily="?attr/fontSemiBold"
-            android:paddingEnd="@dimen/learn_drop_down_icon_margin"
-            android:text="@string/label_my_courses"
-            android:textColor="@color/primaryDarkColor"
-            android:textSize="@dimen/edx_large"
-            tools:ignore="RtlSymmetry" />
-
-        <ImageView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:src="@drawable/ic_drop_down"
-            tools:ignore="ContentDescription" />
-
-    </LinearLayout>
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:animateLayoutChanges="true"
+        android:background="@color/neutralWhite"
+        android:theme="@style/AppTheme.AppBarOverlay">
+
+        <com.google.android.material.appbar.CollapsingToolbarLayout
+            android:id="@+id/collapsingToolbar"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/edx_quadruple_margin"
+            app:layout_scrollFlags="scroll|snap|exitUntilCollapsed">
+
+            <androidx.appcompat.widget.Toolbar
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/edx_button_height"
+                android:layout_gravity="start"
+                android:background="@color/neutralWhite"
+                android:paddingStart="@dimen/container_padding"
```

Comment:
As per designs, please use `16dp` paddings for start/end in toolbar

Reply:
The collapsible toolbar adds some padding as well. We have reduced the actual padding to take care of it.